zerver/openapi/zulip.yaml

```diff
@@ -8665,9 +8668,9 @@ paths:
         UI, and are not included in the user's unread count totals.
 
         **Changes**: Deprecated in Zulip 7.0 (feature level 170). Clients connecting
-        to the newer server should use the `/user_topics` endpoint instead.
+        to the newer server should use the [`/user_topics`](update-user-topic) endpoint instead.
         It will be removed once clients have migrated to use the
-        `/user_topics` endpoint.
+        [`/user_topics`](update-user-topic) endpoint.
```

Comment:
Let's simplify this a bit ...

> **Changes**: Deprecated in Zulip 7.0 (feature level 170). Clients connecting to newer servers should use the [`POST /user_topics`](/api/update-user-topic) endpoint as this endpoint may be removed in a future release.

While we're here can we update the description too?

> [Mute or unmute a topic](/help/mute-a-topic) within a stream that the current user is subscribed to.